src/views/TabView/TabBarBottom.js

```diff
@@ -157,6 +164,30 @@ class TabBarBottom extends React.PureComponent<Props> {
       this.props.getTestIDProps && this.props.getTestIDProps(scene);
     return testIDProps;
   };
+  shouldUseHorizontalTabs() {
+    const { routes } = this.props.navigation.state;
+    const {
+      isLandscape,
+      initialLayout,
+      layout,
+      enableHorizontalTabs,
+    } = this.props;
+    const maxTabBarItemWidth = 125;
+    let tabBarWidth;
+    if (layout && layout.width) tabBarWidth = layout.width;
+    else if (initialLayout && initialLayout.width)
+      tabBarWidth = initialLayout.width;
+    else return isPad;
+    if (!enableHorizontalTabs) return false;
+    if (isPad) {
+      return (
+        canUseHorizontalTabs &&
```

Comment:
Instead of this variable `canUseHorizontalTabs`, use it as the default value of `enableHorizontalTabs` prop, so the user can override it